wallet/src/actors/app/methods.rs

```diff
@@ -140,7 +140,7 @@ impl App {
         &mut self,
         session_id: types::SessionId,
         wallet_id: String,
-    ) -> ResponseActFuture<model::Balance> {
+    ) -> ResponseActFuture<model::WalletBalance> {
```

Comment:
Breaking change :eyes:

Reply:
Yep, it is already merged into Sheikah :fireworks: